src/poly/kzg/params.rs

```diff
@@ -266,72 +227,45 @@ impl<E: Engine + Debug> ParamsKZG<E> {
 // TODO: see the issue at https://github.com/appliedzkp/halo2/issues/45
 // So we probably need much smaller verifier key. However for new bases in g1 should be in verifier keys.
 /// KZG multi-open verification parameters
-pub type ParamsVerifierKZG<E> = ParamsKZG<E>;
+#[derive(Debug)]
+pub struct ParamsVerifierKZG<E: Engine> {
+    pub(crate) s_g2: E::G2Affine,
+}
```

Comment:
Is `k` not needed here?
The PLONK verifier needs `k` does it get it from the PLONK `vk`?

Reply:
It gets it from the `vk`, indeed. Which contains the domain.